$NetBSD: patch-aa,v 1.2 2008/01/09 14:10:06 martti Exp $

The default clamd socket as defined in pkgsrc/mail/clamav/files/clamd.sh

--- doc/clamsmtpd.conf.orig	2006-04-06 16:42:26.000000000 +0200
+++ doc/clamsmtpd.conf
@@ -29,7 +29,7 @@ OutAddress: 10026
 #Listen: 0.0.0.0:10025
 
 # The address clamd is listening on
-#ClamAddress: /var/run/clamav/clamd
+#ClamAddress: /tmp/clamd
 
 # A header to add to all scanned email
 #Header: X-Virus-Scanned: ClamAV using ClamSMTP
